### ISSUE TITLE
intel-mkl package failing to install during ubuntu builds

### ISSUE THREAD

**Issue by aaronmarkham** [apache/incubator-mxnet#16300]
## Description
CI builds are failing due to an issue with an MKL package.
```
+ apt-get install -y intel-mkl-2019.1-053
```
Failure here: http://jenkins.mxnet-ci.amazon-ml.com/job/restricted-website-build-master/59/
```
[91mE: There were unauthenticated packages and -y was used without --allow-unauthenticated
[0mThe command '/bin/sh -c /work/ubuntu_mkl.sh' returned a non-zero code: 100
```
and more info...
```
W: GPG error: https://apt.repos.intel.com/mkl all InRelease: The following signatures were invalid: KEYEXPIRED 1569575501

W: The repository 'https://apt.repos.intel.com/mkl all InRelease' is not signed.
```

**Comment by lanking520** [comment]
@PatricZhao @TaoLv Not sure if this is Intel related. Could you please take a look?

**Comment by lanking520** [comment]
PR merged//

**Comment by TaoLv** [comment]
Thank you @lanking520 @aaronmarkham . I will take a look.

**Comment by vpirogov** [comment]
This is an issue with Intel MKL's apt repository. Intel is aware of the problem and working on the fix. See [relevant topic at Intel MKL forum](https://software.intel.com/en-us/forums/intel-math-kernel-library/topic/821365).

**Comment by vpirogov** [comment]
It turns out that GPG key used to sign packages in apt channel has expired. The issue can be resolved by [updating the key](https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo).

**Comment by lanking520** [comment]
@TaoLv I saw someone is trying to solve this problem, but still seemed not working. Is there an ETA for this? https://software.intel.com/en-us/forums/intel-math-kernel-library/topic/821365

**Comment by TaoLv** [comment]
Sorry I’m on holiday and cannot verify it with mobile phone. But I was told that it was already resolved. Could you please reinstall the key and try the steps in https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo again?

发自我的iPhone

在 2019年10月3日，上午1:31，Lanking <notifications@github.com> 写道：

﻿

@TaoLv<https://github.com/TaoLv> I saw someone is trying to solve this problem, but still seemed not working. Is there an ETA for this? https://software.intel.com/en-us/forums/intel-math-kernel-library/topic/821365

**Comment by TaoLv** [comment]
Closed via #16304 and #16369
